base/linalg/blas.jl

```diff
@@ -1312,12 +1311,12 @@ for (fname, elty, relty) in ((:zherk_, :Complex128, :Float64),
            end
            k  = size(A, trans == 'N' ? 2 : 1)
            ccall((@blasfunc($fname), libblas), Void,
-                 (Ptr{UInt8}, Ptr{UInt8}, Ptr{BlasInt}, Ptr{BlasInt},
-                  Ptr{$relty}, Ptr{$elty}, Ptr{BlasInt}, Ptr{$relty},
-                  Ptr{$elty}, Ptr{BlasInt}),
-                 &uplo, &trans, &n, &k,
-                 &α, A, &max(1,stride(A,2)), &β,
-                 C, &max(1,stride(C,2)))
+                 (Ref{UInt8}, Ref{UInt8}, Ref{BlasInt}, Ref{BlasInt},
+                  Ref{$relty}, Ref{$elty}, Ref{BlasInt}, Ref{$relty},
+                  Ref{$elty}, Ref{BlasInt}),
+                  uplo, trans, n, k,
+                  α, A, max(1,stride(A,2)), &β,
```

Comment:
Missed `&` here? Nanosoldier logs complained about this so probably the reason for the benchmark failure.